src/status_im2/contexts/activity_center/events.cljs

```diff
@@ -119,10 +119,15 @@
                      accepted
                      (not dismissed))
                 (toasts/upsert cofx
-                               {:icon       :placeholder
-                                :icon-color colors/primary-50-opa-40
+                               {:user {:full-name         name
+                                       :profile-picture   (:identicon message)
+                                       :status-indicator? true
+                                       :online?           nil
+                                       :size              :small
+                                       :ring?             false}
+                                :icon-color colors/success-50-opa-40
```

Comment:
I believe we can remove the `:icon-color` as we are not displaying any icon in the toast

Reply:
fixed